TelCoColorCoder.cpp

```diff
@@ -0,0 +1,94 @@
+/**************************************************************************************
+* @file        : TelCoColorCoder.cpp
+* @brief       : Contains Implementation for Color Code Class and Utility functions
+*
+**************************************************************************************/
+#include <iostream>
+#include <assert.h>
+
+#include "TelCoColorCoder.h"
+using namespace TelCoColorCoder;
+
+
+const char* MajorColorNames[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* MinorColorNames[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
+const int TelCoColorCoder::numberOfMajorColors = sizeof(MajorColorNames) / sizeof(MajorColorNames[0]);
+const int TelCoColorCoder::numberOfMinorColors = sizeof(MinorColorNames) / sizeof(MinorColorNames[0]);
+
+
+/**
+ * Description     : ColorPair : Constructor
+ *
+ */
+ColorPair::ColorPair(MajorColor major, MinorColor minor)
+:majorColor(major),
+ minorColor(minor)
+{
+}
+
+/**
+ * Description     : ~ColorPair : Destructor
+ *
+ */
+ColorPair::~ColorPair()
+{
+}
+
+/**
+ * Description     : getMajorColor : To get the Major Color Value
+ *
+ */
+MajorColor ColorPair::getMajorColor()
+{
+	return majorColor;
+}
+
+/**
+ * Description     : getMinorColor : To get the Minor Color Value
+ *
+ */
+MinorColor ColorPair::getMinorColor()
+{
+	return minorColor;
+}
+
+/**
+ * Description     : ToString : To get Minor and Major Color String
+ *
+ */
+std::string ColorPair::ToString()
```

Comment:
Why is this not a Get... function, unlike the other two? It is also getting a string...